### PR TITLE
feat(typography): add stories, modify css

### PR DESCRIPTION
**Pull request by robbieatacato**
## Solve
https://github.com/nl-design-system/denhaag/issues/607

## Purpose
Heading and paragraph:
- html
- css

#closes https://github.com/nl-design-system/denhaag/issues/607

### CODE REVIEW

**Comment by robbieatacato** [comment]
@bddjong I fixed the merge conflict with the main branch.
Can you check this pr, this one is essential to multiple components.

**Comment by bddjong** [comment]
We're not sure where the detail text would be used. I'd like to avoid adding unused components to the library. Blocking this PR for now until there's a use case documented in the issue.

**Comment by robbieatacato** [comment]
> We're not sure where the detail text would be used. I'd like to avoid adding unused components to the library. Blocking this PR for now until there's a use case documented in the issue. There's a single use in the posttype label, but I don't think a single text variation should be a reason to create a separate typography component. @mjcarsjens @Robbert

Any progression?

File: components/Typography/src/paragraph.css [hunk 1]
```diff
@@ -20,3 +20,17 @@
...
+  text-transform: uppercase;
+}
+
+.denhaag-paragraph--detail-small {
+  font-size: var(--denhaag-typography-scale-s-font-size, 0.75rem);
+}
+
+.denhaag-paragraph--detail-extra-small {
+  font-size: var(--denhaag-typography-scale-xs-font-size, 0.625rem);
+}
```
Comment (bddjong) [inline components/Typography/src/paragraph.css:36]: These variations can probably be removed, using a default `--detail` font-size of 12px (.75rem)

**Review by bddjong — CHANGES_REQUESTED** [review]
The discussion has been resolved for now. See comment on fix

File: components/Typography/src/paragraph.css (hunk 1 above)
Comment (robbieatacato) [reply components/Typography/src/paragraph.css]: I refactored it, removed the extra small detail story + re-used the posttype-label tokens.

**Review by bddjong — APPROVED** [review]

**Pull request merged**